roles/variables/vars/main.yml

```diff
@@ -202,6 +202,14 @@ kafka_controller_properties:
     enabled: "{{kafka_controller_sasl_protocol=='kerberos'}}"
     properties:
       sasl.mechanism.controller.protocol: GSSAPI
+  kraft_scram_512:
+    enabled: "{{'SCRAM-SHA-512' in kafka_sasl_enabled_mechanisms}}"
+    properties:
+      super.users: "{% for user in sasl_scram_users_final.values() %}User:{{ user.principal }};{% endfor %}"
```

Comment:
why are we adding super.users here?

Reply:
This was added considering one controller scenario but we have discussed to not add this. Will remove this.